ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -138,6 +139,13 @@ pub struct User {
     pub created_at: DateTime<Utc>,
 }
 
+#[derive(Serialize, Deserialize, GraphQLObject)]
```

Comment:
there's no need to be a `GraphQLObject`, at least for now.

Reply:
after a second thought - since we're creating / consuming SchedulerRunMeta in webserver anyway, would it be better if we simply make this a new table (e.g job_runs, with following columns: `job`, `start_ts`, `end_ts`, `success`, `stdin`, `stderr`), what do you think?

Reply:
let us go with db then